schema.graphql

```diff
@@ -65,3 +65,35 @@ type Extrinsic @entity {
   extrinsic_hash: String
   spec_version_id: Int!
 }
+
+type TokenHolder @jsonField {
+  did: String!
+  amount: String!
+}
+
+type TokenDocument @jsonField {
+  name: String!
+  link: String!
+}
+
+type Token @entity {
```

Comment:
In the SDK we are trying to use the term `Asset` instead of `Token`.

Would you mind using that language here as well, i.e. `Token` -> `Asset`?

Reply:
Yeah that better, will update.